### ISSUE TITLE
[Question] Confictional doesn't work for object

### ISSUE THREAD

**Issue by adhonay** [formio/angular#985]
To reproduce:
https://formio.github.io/formio.js/app/sandbox
Sandbox: 64c3c4d1c27319e1a1b2b486

I need to send the select value in object format { id, label }, but when I create a condition for the text field, to display it when the select value is 1, it doesn't work. This only works for when sending the pure value.

How could I make a conditional or logic via interface in the modal to hide a field, but sending an object?

No conditionals or advanced logic, just a simple interface.

**Comment by HannaKurban** [comment]
You can try the following conditions for the TextField component
```
"conditions": [
      {
        "component": "select",
        "operator": "isEqual",
        "value": {
          "id": 1,
          "label": "Item 1"
        }
      }
    ]
```

**Comment by adhonay** [comment]
@HannaKurban How to do this via interface in a friendly way?

![image](https://github.com/formio/angular/assets/14100568/1118e9c2-7a0b-443a-b6f2-e6c34850fc38)

**Comment by HannaKurban** [comment]
@adhonay The new simple Condition functionality was included in the 8.1.0 release as part of the premium module (Show/Hide UI using simple dropdown fields)

**Comment by adhonay** [comment]
@HannaKurban So in my case, I'm creating a custom dropdown component and I'm sending the value in the object pattern {id, label} . I won't be able to do simple conditionals? if you pass an object. Because when I send only the raw value of the id, it works.
